system/lib/compiler-rt/lib/ubsan_minimal/ubsan_minimal_handlers.cc

```diff
@@ -1,12 +1,3 @@
-/**
- * Copied from compiler-rt.
- * Last changed revision: 351178
- * Last changed date: 2019-01-15.
- *
- * Changes:
- *   * switched to using emscripten_return_address instead of
-       __builtin_return_address. clang currently rejects the latter on wasm.
```

Comment:
why remove this list of changes?

Reply:
Oh sorry.    

1. The information about which version this is take from already exists in system/lib/compiler-rt/lib/ubsan_minimal/readme.txt (I don't know what somebody chose to duplicate that in this file).

2. The actual change in question is no actually here in the file.  This file contains __builtin_return_address and does not contain emscripten_return_address.. .in direct contradiction to the comment.

Reply:
In other words this comment is redundant, inconsistent (with other mods in other libraries) and inaccurate :)

Reply:
Heh, sounds good then...